### PR TITLE
add Pauper EDH to enums

### LINKED ISSUE #1753: Add Pauper Commander Support
**Is your feature request related to a problem? Please describe.**
Pauper Commander (Pauper EDH) is not available
image

But it is available in spell table:
image

**Describe the solution you'd like**
`/lfg format` works with `Pauper EDH`, spawning 

**Describe alternatives you've considered**
None, sorry

**Additional context**
Pauper Commander is also supported at the competitive level:
https://cpdh.guide/

### PR DESCRIPTION
**Pull request by camclark**
Pauper EDH and Pauper Commander are the same thing.
4 Player, 30 Life format
https://pdhhomebase.com/rules/

I couldn't see health in your repo, so I think that's handled by SpellTable...
Maintainers feel free to edit where I've made mistakes.

**Description**
I've been playing competitive pauper commander, and the community have been using your bot to create games.
Since we've got to modify the health to 30 this saves us a little bit of hassle.

- Resolves #1753 

**Checklist**

- [ ] Add tests for these changes
- [X] Test coverage is not decreased
- [X] Entire test suite passes

### CODE REVIEW

File: src/spellbot/enums.py [hunk 1]
```diff
@@ -114,4 +115,5 @@ def __str__(self) -> str:
     GameFormat.VINTAGE,
     GameFormat.SEALED,
     GameFormat.BRAWL_TWO_PLAYER,
+    GameFormat.PAUPER_EDH,
```
Comment (lexicalunit) [inline src/spellbot/enums.py:118]: What do you think about moving this to after CEDH? That way it's closer to all the other EDH formats.

File: src/spellbot/enums.py [hunk 2]
```diff
@@ -88,7 +88,8 @@ def __str__(self) -> str:
     OATHBREAKER = 4
     DUEL_COMMANDER = 2
     CEDH = 4, "cEDH"
-    ARCHENEMY = 4
+    ARCHENEMY = 4,
```
Comment (lexicalunit) [inline src/spellbot/enums.py:91]: No need for the comma here, we can remove that.
```suggestion
    ARCHENEMY = 4
```

Comment (camclark) [reply src/spellbot/enums.py:91]: Ah! Good catch.

File: src/spellbot/enums.py (hunk 1 above)
Comment (camclark) [reply src/spellbot/enums.py:118]: Good point. :)

**Pull request merged**